### PR TITLE
Improve naming and Add IsIPv4MappedToIPv6 to example

### PR DESCRIPTION
**Pull request by KuanWeiTW**
1. Add IsIPv4MappedToIPv6 to example for better support

2. Do changes below because they are doing ip-filtering
ClientIdCheckFilter -> ClientIpCheckFilter
ClientIdCheckPageFilter  -> ClientIpCheckPageFilter

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
I can't see the difference between these two 
`ClientIdCheckFilter -> ClientIpCheckFilter`
- so it's probably not worth changing. We should keep changes to a minimum to make review easier

File: aspnetcore/security/ip-safelist.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The middleware parses the string into an array and looks for the remote IP addre
 
 If you want a safelist only for specific controllers or action methods, use an action filter. Here's an example: 
 
-[!code-csharp[](ip-safelist/samples/2.x/ClientIpAspNetCore/Filters/ClientIdCheckFilter.cs)]
+[!code-csharp[](ip-safelist/samples/2.x/ClientIpAspNetCore/Filters/ClientIpCheckFilter.cs)]
```
Comment (Rick-Anderson) [inline aspnetcore/security/ip-safelist.md]: Backout name change. It's just a bunch of noise.

**Comment by Rick-Anderson** [comment]
I get it now. Changed `Id` to `IP`
You can keep that change.

**Comment by Rick-Anderson** [comment]
@KuanWeiTW  lets do this in two steps. Backout name change for now. You can add name change back in another commit or another PR. That will make it easier to review code changes.

**Comment by KuanWeiTW** [comment]
OK

**Comment by KuanWeiTW** [comment]
@Rick-Anderson , I created 2 new PR ( #15304 #15305 ) for splitting changes. Please help to review them. Thanks!

**Comment by Rick-Anderson** [comment]
@KuanWeiTW  really appreciate you doing that. Normally I wouldn't ask. Can we close this PR now?

**Pull request closed without merging**